### PR TITLE
Beego.me make new trivial rule.

### PR DESCRIPTION
**Pull request by jish**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
Thanks. Please:

Note as `Timeout`:
- [ ] www.beego.me
- [ ] fanyi.beego.me

`Invalid certificate`:
- [ ] zh.beego.me

Also:
- [ ] Add targets `www`, `fanyi`, `zh` and rewrite them to `^`

**Comment by jish** [comment]
I added comments nothing the problematic domains. I'm not sure I know how to rewrite to `^` 😞

**Comment by gloomy-ghost** [comment]
Thanks, you can either choose separately list problematic subdomains like:

```
<!--
	Invalid certificate:
		- zh.beego.me

	Timeout:
		- www.beego.me
		- fanyi.beego.me
-->
```

or use the comments provided by make-trivial like:

```
<!--
	Problematic subdomains:
		- www.beego.me   (t)
		- fanyi.beego.me (t)
		- zh.beego.me    (m)

	m: certificate mismatch
	t: timeout on https
-->
```

Note that we would like to sort subdomains in alphabetical order starting from the top level domain at the right reading left, moving ^ and www to the top of their group. For example:

```
example.com
www.example.com
a.example.com
www.a.example.com
b.a.example.com
b.example.com

example.net
www.example.net
a.example.net
```

**Comment by gloomy-ghost** [comment]
For complex rules, in this case you can use
```
	<rule from="^http://(www|fanyi|zh)\.beego\.me/" 
		to="https://$1.beego.me" />
```

**Comment by jish** [comment]
Forgive me if I misunderstood your comment, but I'm assuming that `^` means the domain.

> "Add targets `www`, `fanyi`, `zh` and rewrite them to `^`"

So, I've explicitly listed out each rule to clarify the intent. If that is not what we want then I can update the rules as necessary 😄

File: src/chrome/content/rules/Beego.me.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<ruleset name="Beego.me">
+	<target host="beego.me" />
+	<target host="www.beego.me" />
+
+	<target host="fanyi.beego.me" />
+	<target host="zh.beego.me" />
+
+	<rule from="^http://www\.beego\.me/" to="https://beego.me/" />
+	<rule from="^http://fanyi\.beego\.me/" to="https://beego.me/" />
+	<rule from="^http://zh\.beego\.me/" to="https://beego.me/" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Beego.me.xml]: Please squash these rules into one rule since they are all redirecting to `^`.

**Comment by gloomy-ghost** [comment]
Oops, just updated https://github.com/EFForg/https-everywhere/pull/7750#issuecomment-265072630

**Comment by jish** [comment]
I combined the three rules that have the same destination into one rule.

**Comment by gloomy-ghost** [comment]
Thanks, merged.

**Pull request merged**